modal/exception.py

```diff
@@ -88,6 +88,12 @@ class DeprecationError(UserWarning):
     # Overloading it to evade the default filter, which excludes __main__.
 
 
+class AppStopped(Error):
```

Comment:
Maybe it should inherit from `BaseException`? Similar to other "exit" exceptions: https://docs.python.org/3/library/exceptions.html

Reply:
Good point!

Reply:
But on the other hand, it's not always meant to exit the interpreter as it's only an indicator that the remote app has stopped. Like in a script you wouldn't necessarily want that to exit your script (and if not a script, then it's our CLI so we can handle it as we like?)

Reply:
oh got it, this is not in the app that gets stopped itself